ambry-tools/src/main/java/com.github.ambry.store/HardDeleteVerifier.java

```diff
@@ -11,7 +11,7 @@
  * distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
Can't change it to public as the class name is different from file name. @pnarayanan : Do you remember the reason why is it so. Or can I go ahead and change the class name